py/configu/commands/eval_command.py

```diff
@@ -24,90 +24,92 @@ class EvaluatedConfigOrigin(Enum):
     EmptyValue = "EMPTY_VALUE"
 
 
-EvalCommandReturnContext = TypedDict(
-    "EvalCommandReturnContext",
-    {
-        "store": str,
-        "set": str,
-        "schema": str,
-        "key": str,
-        "cfgu": Union[ConfigSchemaContentsValue, Cfgu],
-    },
-)
+class EvalCommandReturnContext(TypedDict):
+    store: str
+    set: str
+    schema: str
+    key: str
+    cfgu: Union[ConfigSchemaContentsValue, Cfgu]
 
-EvalCommandReturnResult = TypedDict(
-    "EvalCommandReturnResult",
-    {
-        "origin": EvaluatedConfigOrigin,
-        "source": str,
-        "value": str,
-    },
-)
 
-EvalCommandReturnValue = TypedDict(
-    "EvalCommandReturnValue",
-    {
-        "context": EvalCommandReturnContext,
-        "result": EvalCommandReturnResult,
-    },
-)
-EvalCommandReturn = Dict[str, EvalCommandReturnValue]
-
-EvalCommandParameters = TypedDict(
-    "EvalCommandParameters",
-    {
-        "store": ConfigStore,
-        "set": ConfigSet,
-        "schema": ConfigSchema,
-        "configs": Optional[Dict[str, str]],
-        "validate": Optional[bool],
-        "previous": Optional[EvalCommandReturn],
-    },
-    total=False,
-)
+class EvalCommandReturnResult(TypedDict):
+    origin: EvaluatedConfigOrigin
+    source: str
+    value: str
+
+
+class EvalCommandReturnValue(TypedDict):
+    context: EvalCommandReturnContext
```

Comment:
this should be aligned with the ts sdk the eval now returns a flat object that the export then transforms to the result

Reply:
fixed. please notice on data3 EvalCommand, even though i'm passing configs overrides for `GREETING` it doesn't override it. this is because this key is from the previous and current schema. is this intended? 
![image](https://github.com/configu/configu/assets/129258356/893a1b42-93aa-4282-bdea-e8c4bb864fa1)

Reply:
@RonConfigu Im not sure #171 is a bug we should discuses it.